packages/cli/src/commands/redis/cli.ts

```diff
@@ -105,34 +103,37 @@ async function redisCLI(uri: URL, client: Writable): Promise<void> {
   })
 }
 
-async function bastionConnect(uri: URL, bastions: string, config: Record<string, unknown>, preferNativeTls: boolean) {
-  const tunnel: Client = await new Promise(resolve => {
-    const ssh2 = new Client()
-    resolve(ssh2)
```

Comment:
Maybe this line was resolving immediately before the `ssh2` connection was ready to handle traffic. It doesn't look as if it should be here at all, but I'm OK with undoing the refactor to fix the current issue and maybe retry a refactor when moving to a new Node.js version.

Reply:
You were right, removing that line fixed it, thank you!! I'm going to close this and submit a new PR.